src/olympia/reviewers/views.py

```diff
@@ -136,6 +137,101 @@ def beta_signed_log(request):
+            ugettext('New Add-ons ({0})').format(
+                ViewFullReviewQueue.objects.count()),
+            reverse('reviewers.queue_nominated')
+        ), (
+            ugettext('Add-on Updates ({0})').format(
+                ViewPendingQueue.objects.count()),
+            reverse('reviewers.queue_pending')
+        ), (
+            ugettext('Performance'),
+            reverse('reviewers.performance')
+        ), (
+            ugettext('Add-on Review Log'),
+            reverse('reviewers.reviewlog')
+        ), (
+            ugettext('Signed Beta Files Log'),
+            reverse('reviewers.beta_signed_log')
+        )]
+    if acl.action_allowed(request, amo.permissions.ADDONS_POST_REVIEW):
+        sections[ugettext('Auto-Approved Add-ons')] = [(
+            ugettext('Auto Approved Add-ons ({0})').format(
+                AutoApprovalSummary.get_auto_approved_queue().count()),
+            reverse('reviewers.queue_auto_approved')
+        ), (
+            ugettext('Performance'),
+            reverse('reviewers.performance')
+        ), (
+            ugettext('Add-on Review Log'),
+            reverse('reviewers.reviewlog')
+        )]
+    if acl.action_allowed(request, amo.permissions.ADDONS_CONTENT_REVIEW):
+        sections[ugettext('Content Review')] = [(
+            ugettext('Content Review ({0})').format(
+                AutoApprovalSummary.get_content_review_queue().count()),
+            reverse('reviewers.queue_content_review')
+        ), (
+            ugettext('Performance'),
+            reverse('reviewers.performance')
+        )]
+    if acl.action_allowed(request, amo.permissions.THEMES_REVIEW):
+        sections[ugettext('Lightweight Themes')] = [(
+            ugettext('New Themes ({0})').format(
+                Persona.objects.filter(
+                    addon__status=amo.STATUS_PENDING).count()),
+            reverse('reviewers.themes.list')
+        ), (
+            ugettext('Themes Updates ({0})').format(
+                RereviewQueueTheme.objects.count()),
+            reverse('reviewers.themes.list_rereview')
+        ), (
+            ugettext('Flagged Themes ({0})').format(
+                Persona.objects.filter(
+                    addon__status=amo.STATUS_REVIEW_PENDING).count()),
+            reverse('reviewers.themes.list_flagged')
+        ), (
+            ugettext('Themes Review Log'),
+            reverse('reviewers.themes.logs')
+        ), (
+            ugettext('Deleted themes Log'),
+            reverse('reviewers.themes.deleted')
+        )]
+    if acl.action_allowed(request, amo.permissions.RATINGS_MODERATE):
+        sections[ugettext('User Ratings Moderation')] = [(
+            ugettext('Ratings Awaiting Moderation ({0})').format(
+                Rating.objects.all().to_moderate().count()),
+            reverse('reviewers.queue_moderated')
+        ), (
+            ugettext('Moderated Review Log'),
+            reverse('reviewers.eventlog')
```

Comment:
`def eventlog` was changed to `@any_reviewer_or_moderator_required` but we only show it for `RATINGS_MODERATE` users?  Deliberate?

Reply:
Mmm good point. I think I'll change `evenlog` to only require ratings moderators. Every legacy reviewer should have been included in the group that only has the ratings moderation permission by now.